### PR TITLE
[Feature: #154589791] Deactivate a user's account.

### PR DESCRIPTION
**Pull request by oreoluwade**
#### What does this PR do?
- Enables User account deactivation.

#### Description of Task to be completed?
- Create mutation for deactivating a user account.
- Write test cases for different account deactivation scenarios.
- Update Payment test suite (to catch uncovered lines).
- Prevent Login for deactivated user.
- Prevent details update for deactivated user.

#### How should this be manually tested?
- Check out into the `ft-deactivate-user-154589791` branch.
- Run the `deactivateAccount` mutation on `graphiql` for: A. A fresh deactivation B. An already deactivated account. C. A non-existent account
- Ensure that you get the correct messages/results every time.
- Run tests with `coverage run --source=food_basket --omit */snapshots/* manage.py test`
- Ensure all lines are covered with `coverage report -m`

#### Any background context you want to provide?
- When a user no longer has the need for their account, there should be an option to deactivate said account (not delete, so that records can be kept, and the user has the option to reactivate the account in the future).

#### What are the relevant pivotal tracker stories?
- [#154589791](https://www.pivotaltracker.com/story/show/154589791)

### CODE REVIEW

File: food_basket/tests/tests_008_payment_schema.py [hunk 1]
```diff
@@ -170,6 +170,19 @@ def test_update_payment_not_admin(self):
               '''
         self.assertMatchSnapshot(client.execute(mutation))
 
+    # Test getting all payments
+    def test_retrieve_all_payments(self):
```
Comment (omorilewa) [inline food_basket/tests/tests_008_payment_schema.py]: @oreoluwade Is there a reason you are working on payment. Is it directly or indirectly related to your story?

**Comment by omorilewa** [comment]
@oreoluwade I do not think it should be possible to update the details of a deactivated account.

File: food_basket/tests/tests_008_payment_schema.py (hunk 1 above)
Comment (oreoluwade) [reply food_basket/tests/tests_008_payment_schema.py]: @omorilewa while running tests, I found that the payment schema was not fully tested. I don't know how this happened, and I believe that I included that this included in the PR description. 😏

Comment (omorilewa) [reply food_basket/tests/tests_008_payment_schema.py]: I saw the PR description. Thanks for pointing it to me again. Except it affected your coverage, I think it should have been a bug (a different story).

Comment (oreoluwade) [reply food_basket/tests/tests_008_payment_schema.py]: Okay, @omorilewa. Point noted.

Comment (omorilewa) [reply food_basket/tests/tests_008_payment_schema.py]: @oreoluwade Nope. Just to bring it to your notice. This may slide, another is unlikely to. 😬 🙇‍♀️

**Comment by omorilewa** [comment]
@oreoluwade Do you intend to add this(update not allowed on a deactivated account) to this PR?

File: food_basket/tests/tests_008_payment_schema.py (hunk 1 above)
Comment (oreoluwade) [reply food_basket/tests/tests_008_payment_schema.py]: @omorilewa. Thanks for letting it slide. :smirk:

It is a weakness for me to leave things undone, that could easily be done. I shall try to work on this weakness.

**Comment by oreoluwade** [comment]
@omorilewa Yes. I am looking to do that. Will work on that soonest.

**Comment by oreoluwade** [comment]
@omorilewa I have now Implemented the disallowing of update as well as Login for a deactivated user.

**Pull request merged**